bundle/manifests/numaresources-operator.clusterserviceversion.yaml

```diff
@@ -65,6 +65,11 @@ spec:
       - description: Group of Nodes to enable RTE on
         displayName: Group of nodes to enable RTE on
         path: nodeGroups
+      - description: Set the topology info refresh period. Use explicit 0 to disable.
```

Comment:
Why is only the refresh period here and not `infoRefreshMode`? The refresh period is linked to `infoRefreshMode` and depends on whether `Periodic` or `Evented` so was expecting both here.

Reply:
good catch, I don't have a answer, need to check.

Reply:
the root cause was a typo in the `operator-sdk` annotation:
```
-       //+operator-sdk:csv:customresourcedefinition:type=spec,displayName="Topology info mechanism setting",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
+       //+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Topology info mechanism setting",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
```